src/content/en/basic-workflows/z-image.md

```
@@ -0,0 +1,84 @@
+summary: "Image generation with Z-Image"
+permalink: "/{{ lang }}/{{ section }}/{{ slug }}/"
+hero:
+  image: "https://i.gyazo.com/39cddc1debeff5423090f2fe87e5b038.png"
+tags: []
+---
+
+## What is Z-Image?
+
+Z-Image is a **family of image generation models** by Alibaba / Tongyi-MAI.
+
+![](https://gyazo.com/126c0d5ef1364355014fdd7e3288825c){gyazo=image}
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)